docs/getting_started.rst

```diff
@@ -91,3 +91,35 @@ To check your installed version of `mmpy_bot`, simply open a Python interpreter
 
         import mmpy_bot
         print(mmpy_bot.__version__)
+
+
+Integration with Django
+-----------------------
+
+You can create a ``django`` command as follows:
+
+    from django.core.management.base import BaseCommand
+
+    from mmpy_bot import Bot, Settings
+
+
+    class Command(BaseCommand):
+
+        def handle(self, **options):
+            b = Bot(settings=Settings())
+            b.run()
+
+
+Modify ``manage.py``::
+
+    #!/usr/bin/env python
+
+    import os
+    import sys
+
+
+    if __name__ == "__main__":
+        os.environ.setdefault("DJANGO_SETTINGS_MODULE", "project.settings")
+        from django.core.management import execute_from_command_line
+
+        execute_from_command_line(sys.argv)
```

Comment:
Can someone have a look at this? I have zero Django experience and this example probably doesn't work, but I'm not sure how Django works in the first place.

Reply:
I look at this but don't really know what I am looking at 😸 Honestly I am not sure we need to include the Django integration section. What is the actual use case for that?

Reply:
Yeah fair point, I wouldn't know either... I'll remove it altogether 👍🏼